lib/internal/modules/esm/translators.js

```diff
@@ -259,7 +259,6 @@ translators.set('builtin', async function builtinStrategy(url) {
 
 // Strategy for loading a JSON file
 translators.set('json', async function jsonStrategy(url, source) {
-  emitExperimentalWarning('Importing JSON modules');
```

Comment:
I'm unsure we want to remove the warning just yet.

Reply:
You think we should keep importing of JSON as experimental even after it’s unflagged? Why?

Reply:
The same strategy was used for ESM when we removed `--experimental-modules` we kept the warning for a while. Also worth noting that import assertions are still flagged in V8.

Reply:
Okay. Maybe we can target 18.0.0 for dropping the warning.